bundler/dependabot-bundler.gemspec

```diff
@@ -42,4 +42,5 @@ Gem::Specification.new do |spec|
       spec.files << path unless File.directory?(path)
     end
   end
+  spec.metadata["rubygems_mfa_required"] = "true"
```

Comment:
I don't think we would want to enable MFA here 
https://guides.rubygems.org/mfa-requirement-opt-in/

Reply:
This line is also included in the other `dependabot-*.gemspec` files

Reply:
Nice catch! I'll remove that and disable the relevant check.

Reply:
Fixed by 1cc36ec31.